src/openrct2-ui/windows/RideList.cpp

```diff
@@ -769,154 +775,141 @@ class RideListWindow final : public Window
-                        currentListPosition, rideRef, [](const Ride& thisRide, const Ride& otherRide) -> bool {
-                            return thisRide.GetTotalQueueLength() <= otherRide.GetTotalQueueLength();
-                        });
-                    break;
-                case INFORMATION_TYPE_QUEUE_TIME:
-                    currentListPosition = SortList(
-                        currentListPosition, rideRef, [](const Ride& thisRide, const Ride& otherRide) -> bool {
-                            return thisRide.GetMaxQueueTime() <= otherRide.GetMaxQueueTime();
-                        });
-                    break;
-                case INFORMATION_TYPE_RELIABILITY:
-                    currentListPosition = SortList(
-                        currentListPosition, rideRef, [](const Ride& thisRide, const Ride& otherRide) -> bool {
-                            return thisRide.reliability_percentage <= otherRide.reliability_percentage;
-                        });
-                    break;
-                case INFORMATION_TYPE_DOWN_TIME:
-                    currentListPosition = SortList(
-                        currentListPosition, rideRef, [](const Ride& thisRide, const Ride& otherRide) -> bool {
-                            return thisRide.downtime <= otherRide.downtime;
-                        });
-                    break;
-                case INFORMATION_TYPE_GUESTS_FAVOURITE:
-                    currentListPosition = SortList(
-                        currentListPosition, rideRef, [](const Ride& thisRide, const Ride& otherRide) -> bool {
-                            return thisRide.guests_favourite <= otherRide.guests_favourite;
-                        });
-                    break;
-                case INFORMATION_TYPE_EXCITEMENT:
-                    currentListPosition = SortList(
-                        currentListPosition, rideRef, [](const Ride& thisRide, const Ride& otherRide) -> bool {
-                            return thisRide.excitement <= otherRide.excitement;
-                        });
-                    break;
-                case INFORMATION_TYPE_INTENSITY:
-                    currentListPosition = SortList(
-                        currentListPosition, rideRef, [](const Ride& thisRide, const Ride& otherRide) -> bool {
-                            return thisRide.intensity <= otherRide.intensity;
-                        });
-                    break;
-                case INFORMATION_TYPE_NAUSEA:
-                    currentListPosition = SortList(
-                        currentListPosition, rideRef, [](const Ride& thisRide, const Ride& otherRide) -> bool {
-                            return thisRide.nausea <= otherRide.nausea;
-                        });
-                    break;
-            }
-            listIndex++;
+            RideListEntry entry{};
+            entry.Id = rideRef.id;
+            entry.Name = rideRef.GetName();
+
+            _rideList.push_back(entry);
```

Comment:
```suggestion
            _rideList.push_back(std::move(entry));
```

It's a nitpick but it turns a string copy into a string move, so for a performance oriented PR it's always welcome.

Reply:
Good point